wallet/tests/unit/useCases/CommitmentsBackup/import.test.js

```diff
@@ -0,0 +1,95 @@
+import indexedDB from 'fake-indexeddb';
+
+import {
+  addObjectStoreToIndexedDB,
+  convertFileToObject,
+  getIndexedDBObjectRowsFromBackupFile,
+} from '../../../../src/useCases/CommitmentsBackup/import';
+
+const mockObject = [
+  {
+    table: 'transactions',
+    rows: [
+      { key: '1', value: { _id: '1', value: '1' } },
+      { key: '2', value: { _id: '2', value: '2' } },
+      { key: '3', value: { _id: '3', value: '3' } },
+    ],
+  },
+  {
+    table: 'commitments',
+    rows: [
+      { key: '1', value: { _id: '1', value: '1' } },
+      { key: '2', value: { _id: '2', value: '2' } },
+      { key: '3', value: { _id: '3', value: '3' } },
+    ],
+  },
+  {
+    table: 'keys',
+    rows: [
+      { key: '1', value: { _id: '1', value: '1' } },
+      { key: '2', value: { _id: '2', value: '2' } },
+      { key: '3', value: { _id: '3', value: '3' } },
+    ],
+  },
+];
+
+describe('Tests about file', () => {
+  test('Should convert a blob file in a object', async () => {
+    const file = new Blob([JSON.stringify(mockObject)], { type: 'text/plain' });
+    expect(file).not.toEqual(mockObject);
```

Comment:
You don't need this, since `file` is an instance of Blob and `mockObject` is an array of objects, it is straightforward to see they aren't equal.

Reply:
Really. I don't know what I left it there. Removed line 39.